src/ape_ethereum/ecosystem.py

```diff
@@ -252,7 +232,7 @@ def get_proxy_info(self, address: AddressType) -> Optional[ProxyInfo]:
         return None
 
     def serialize_transaction(self, transaction: TransactionAPI) -> bytes:
-        return transaction.serialize_transaction()
+        return self.serialize_transaction(transaction)
```

Comment:
This will cause an infinite loop!
We can delete this method if only needs to call the base class.
Otherwise, use `super()`